doc/nrf/gs_installing.rst

```diff
@@ -372,11 +372,7 @@ The Zephyr Software Development Kit (SDK) contains toolchains for each of Zephyr
 It also includes additional host tools, such as custom QEMU and OpenOCD builds.
 
 .. note::
-
-   Set the following environment variables to ensure the Zephyr SDK is detected correctly:
-
-   * :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``zephyr``
-   * :envvar:`ZEPHYR_SDK_INSTALL_DIR` to the path of the Zephyr SDK
+   When updating Zephyr SDK, :ref:`verify the Zephyr SDK variables <zephyr:toolchain_zephyr_sdk_update>`.
```

Comment:
I am worried that users will go to the linked page, see more links listing things to verify, and either click off, or go to those links and not understand that the key thing to check is that the Zephyr toolchain is selected.

A slight addition would prevent this:

```suggestion
   When updating Zephyr SDK, :ref:`verify the Zephyr SDK variables <zephyr:toolchain_zephyr_sdk_update>`. Make sure that the ``zephyr`` toolchain is selected, not ``gnuarmemb``.
```